geppetto.js/geppetto-client/package.json

```diff
@@ -24,35 +25,46 @@
     "publish:yalc": "cd build && yalc publish --push",
     "watch": "nodemon -e js,ts --ignore build --exec \"babel src -d build --extensions '.js,.jsx,.ts,.tsx' && yalc push --changed\""
   },
-  "peerDependencies": {
```

Comment:
I would like to keep the dependencies as peerDependencies. We want to avoid, at the application level, that when we require geppetto-x for the reason y we don't have to install a bunch of packages that are only needed for other reasons.
Essentially we want applications to only install what they really need